test/modules/layers/test_attention.py

```diff
@@ -81,22 +81,32 @@ def test_multi_head_attention(
         full_attn,
     ):
         mha = multihead_attn(1, False, full_attn)
-        q = 2 * torch.ones(1, *input_shape, hidden_dim)
-        k = 2 * torch.ones(1, *input_shape, hidden_dim)
-        v = 2 * torch.ones(1, *input_shape, hidden_dim)
-        actual = mha(q, k, v)
+        qkv = 2 * torch.ones(1, *input_shape, hidden_dim)
```

Comment:
`test_self_multihead_attention`?
might be good to add `test_cross_multihead_attention` with `kv` provided just for the default kwargs?